### ISSUE TITLE
Sprockets error with 0.14.3

### ISSUE THREAD

**Issue by erlingur** [somebee/imba#47]
After updating `imba-source` to 0.14.3 I get a Sprockets error when using Imba with Rails.

```
Sprockets::FileNotFound - couldn't find file 'imba' with type 'application/javascript':
  sprockets (3.4.0) lib/sprockets/directive_processor.rb:182:in `rescue in block in process_directives'
  sprockets (3.4.0) lib/sprockets/directive_processor.rb:179:in `block in process_directives'
  sprockets (3.4.0) lib/sprockets/directive_processor.rb:178:in `process_directives'
  sprockets (3.4.0) lib/sprockets/directive_processor.rb:83:in `_call'
  sprockets (3.4.0) lib/sprockets/directive_processor.rb:68:in `call'
```

**Comment by somebee** [comment]
I have submitted a [pull request](https://github.com/judofyr/imba-source/pull/2) to imba-source now which will hopefully fix this.

**Comment by erlingur** [comment]
Fixed in 0.14.4 :)
